docs/natspec-format.rst

```diff
@@ -154,10 +154,6 @@ to the end-user as:
 
 if a function is being called and the input ``a`` is assigned a value of 10.
 
-Specifying these dynamic expressions is outside the scope of the Solidity
```

Comment:
Hm, would it make sense turning this into an explainer and mentioning it was abandoned?

Reply:
I do not think it makes sense to pollute the documentation with that - it is not really used in the wild - so I think no one cares. Readers of this doc care how to use it and not about the history I think.